### ISSUE TITLE
examples notebook - Plotting fails

### ISSUE THREAD

**Issue by SDX-LV** [pyfar/pyfar#390]
in [examples notebook](https://mybinder.org/v2/gh/pyfar/pyfar/main?filepath=examples%2Fpyfar_demo.ipynb)
the "Plotting" section 2nd and 4th code blocks fail.

`x_power.fft_norm = 'rms'
pf.plot.line.freq(x_power)`
produces no plot.

`ax = pf.plot.time(x_power)
ax.set_xlim(0, 2)`
produces warning.

Actually after these, all `pf.plot` commands do not produce output or completely crash.

## What I Did

Tried to learn pyfar :)

## Labels

 **bug**  **plot**

**Comment by f-brinkmann** [comment]
Unfortunately this is not a pyfar Problem, but seems to be a bug in Matplotlib. Matplotlib should show plots in case an interactive python backend is used - but it currently does not. Until the bug is fixed, one workaround is to import matplotlib using `import matplotlib.pyplot as plt` and using `plt.show()` after each plot.

The examples notebook gets tested and currently passes the tests: https://app.circleci.com/pipelines/github/pyfar/pyfar/421/workflows/21587d5e-9185-4b4b-8aec-389fa2f83074/jobs/2327

**Comment by SDX-LV** [comment]
Thanks,
I literally did what you said: put the `import matplotlib.pyplot as plt` command in the first code block and then added `plt.show()` into every code block that should produce a plot, and Then it works :)
Otherwise the interactive tutorial is practically crippled.

**Comment by f-brinkmann** [comment]
@SDX-LV,  I looked into it a bit

- found an environment where it works without plt.show()
- found an environment where plt.show() is required
- compared them
- found out that the environments that works as expected was newer
- did a fresh environment using `conda create --name tmp python`, `conda activate tmp`, and `pip install pyfar PyQt5 notebook`
- found out that everything works as expected in the new environment

I assume that the bug was fixed in the meantime. Can you or anyone else confirm this?

**Comment by SDX-LV** [comment]
Hi, I don't know. I now clicked on the link provided on https://pypi.org/project/pyfar/
and it took a long time to re-install something on the server side, but in the end when I ran the notebook, again only the first plot is shown and there was also a warning message in cell 16:
`/srv/conda/envs/notebook/lib/python3.10/site-packages/pyfar/plot/_utils.py:128: UserWarning: Attempted to set non-positive left xlim on a log-scaled axis.
Invalid limit will be ignored.
  setter((min(limits[0], low), max(limits[1], high)))
/tmp/ipykernel_57/298374242.py:2: UserWarning: Attempted to set non-positive left xlim on a log-scaled axis.
Invalid limit will be ignored.
  ax.set_xlim(0, 2)`

So overall it is still the same issue. 
I think that the notebook is absolutely outstanding as a tutorial, but it does not show the plots that are supposed to be visible.

Note I use Opera as browser, so in theory it may be different for other browsers?
